content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -193,6 +193,22 @@ When a Pod consumes a PersistentVolume, the GIDs associated with the
 PersistentVolume are not present on the Pod resource itself.
 {{< /note >}}
 
+## Clean up
```

Comment:
@diguage You can use a `{{% capture cleanup %}}` section for this, I think.

Reply:
Let me try it.

Reply:
@sftim 
I study the structure of the document. The `{{% capture cleanup %}}` is used in the `Tutorials` part. The `Tasks` part uses the `{{% capture steps %}}`, and the `Clean up` is the last step of a task. So, I follow the style.